### ISSUE TITLE
[BUG] running model with compute_masks=False causes cv2.resize error because masks=[0]

### ISSUE THREAD

**Issue by fyz11** [MouseLand/cellpose#1200]
I just upgraded to cellpose 4.0.4 today. 

I ran the pretrained 'cyto' model with compute_masks=False, and a diameter value which was not `None`. This led to a cv2.resize error. 
![Image](https://github.com/user-attachments/assets/f3098504-9071-4818-b34e-be9c073e2e78)

Tracing the error it occurs in `models.py` with the `model.eval`. When compute_masks=False, masks=[0]. However, it is the current code is still trying to resize this back to the input image resolution. Please could you fix this? I have drawn an arrow to indicate the line needing to fix - thanks !

![Image](https://github.com/user-attachments/assets/5cdd9f86-38c0-4425-8bcb-fd06356bf1b7)

**Comment by fyz11** [comment]
Sure thing.

I am running Python 3.9.21 in a conda environment, and running cellpose through the code API. 

This is the screenshot of the welcome message. 
![Image](https://github.com/user-attachments/assets/01b7d293-20d0-48cc-9491-40618572a244)

This is a minimal code snippet to reproduce the error. 
```
  import skimage.io as skio 
  import pylab as plt 
  import numpy as np 
  import scipy.ndimage as ndimage
  import skimage.segmentation as sksegmentation 
  import os 
  
  import cellpose
  import cellpose.models as models
  
  
  """
  Load the image
  """
  trainfolder = '/archive/bioinformatics/shared/U54_segment/Public_Datasets_for_CellSegmentation/Cellpose_Dataset/Training/train'
  
  imgfile = os.path.join(trainfolder,
                         '000_img.png')
  img = skio.imread(imgfile)
  
  
  
  """
  This breaks, when diameter=20 and compute_masks=False
  """
  cp_model = models.CellposeModel(gpu=True, pretrained_model='cyto')
  masks, flow, style = cp_model.eval(img,
                                     diameter=60, 
                                     channels=[0,0],
                                     cellprob_threshold=-np.inf,
                                     flow_threshold=0.85, 
                                     compute_masks=False)
  _, flow, prob = flow
  
  plt.figure()
  plt.imshow(prob)
  plt.show()
```

The image I am using is the train/000_img.png from the cellpose training dataset for illustration (downloaded from https://www.[cellpose](https://www.cellpose.org/).org/). The error doesn't matter what the dimensions are. The trigger conditions is a custom value for `diameter` and `compute_masks=False`.


I attach my `pip list` output.
[cellpose4_error_env.txt](https://github.com/user-attachments/files/20399780/cellpose4_error_env.txt)

Thanks for looking into this.
